Toolchains/toolchain_x86_64.cmake

```diff
@@ -0,0 +1,19 @@
+# This is tuned for Ubuntu + gcc
+# Fun Fact: Ubuntu's i386 actually uses the i686 compiler
```

Comment:
The fun fact only needs to be mentioned in the 32-bit toolchain.

Reply:
ha, true